File: Chapter 04/Python/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/models/item.py
class Item:
    """
    ARCHITECTURE NOTE: A simple data entity. The behavior regarding 
    how items are priced and discounted is encapsulated inside the 
    Rich 'Order' model, not here.
    """
    def __init__(self, id: int = 0, price: float = 0.0, quantity: int = 0):
        self.id = id
        self.price = price
        self.quantity = quantity

File: Chapter 04/JS/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/models/item.js
/**
 * ARCHITECTURE NOTE: A simple data entity. The behavior regarding 
 * how items are priced and discounted is encapsulated inside the 
 * Rich 'Order' model, not here.
 */
class Item {
    constructor(id = 0, price = 0.0, quantity = 0) {
        this.id = id;
        this.price = price;
        this.quantity = quantity;
    }
}
module.exports = Item;